src/app/directives/response.tpl.html

```diff
@@ -21,11 +21,11 @@ <h4 class="raml-console-resource-body-heading">
 
   <div ng-if="responseInfo[responseInfo.currentType].example">
     <span>Example:</span>
-    <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[responseInfo.currentType].example)"></code></pre>
+    <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeautifiedExampleRef(responseInfo[responseInfo.currentType].example)"></code></pre>
```

Comment:
Why are we putting `getBeautifiedExample` on the scope at [line 23](https://github.com/mulesoft/api-console/pull/212/files#diff-335dba442edca17b70abb80db21a7fccR23) 

If we're using `getBeautifiedExampleRef` here?